### PR TITLE
Added custom filters in get-instance-info

### PR DESCRIPTION
**Pull request by gshriki**
Added the ability to add custom filter values in the get-instance-info command

### CODE REVIEW

File: Integrations/integration-aws.yml [hunk 1]
```diff
@@ -214,6 +213,17 @@ script:
             urlArgs[filter+'.Value'] = keyName;
             filterid = filterid + 1;
         }
+        if ((customFilters) && (customFilters.length > 0)) {
```
Comment (dorsha) [inline Integrations/integration-aws.yml]: Syntax wise, I would change it to be like this:

```
if (customFilters && customFilters.length > 0) {
        customFilters.split(',').forEach(function (customFilter) {
            filter = 'Filter.' + filterid.toString();
            urlArgs[filter+'.Name'] = customFilter.split("=")[0];
            urlArgs[filter+'.Value'] = customFilter.split("=")[1];
            filterid = filterid + 1;
        });
    }
```

Comment (gshriki) [inline Integrations/integration-aws.yml]: fixed

Comment (gshriki) [inline Integrations/integration-aws.yml]: fixed

File: Integrations/integration-aws.yml [hunk 2]
```diff
@@ -413,3 +424,4 @@ script:
     - name: headers
       description: Headers to display in Human readable format
     description: Get EC2 EBS Volume Information
+hidden: false
```
Comment (meirwah) [inline Integrations/integration-aws.yml]: not needed i think

File: Integrations/integration-aws.yml [hunk 3]
```diff
@@ -4,7 +4,6 @@ commonfields:
 name: Amazon Web Services
 display: Amazon Web Services
 category: IT Services
-system: true
```
Comment (meirwah) [inline Integrations/integration-aws.yml]: you remove this on propuse  ?

Comment (dorsha) [inline Integrations/integration-aws.yml]: It is not needed anymore

Comment (gshriki) [inline Integrations/integration-aws.yml]: the system removed it when I downloaded the script. It's not needed anymore...  (anything that comes from us is automatically tagged as system as far as I know)

**Pull request merged**